src/compiler/parse/utils/html.ts

```diff
@@ -35,8 +35,18 @@ const windows_1252 = [
 	376
 ];
 
+const reg_exp_entity = function (entity_name: string) {
+	if (entity_name.endsWith(';')) {
+		return entity_name;
+	}
+	return String.raw`${entity_name}\b`;
+};
+
+const reg_exp_num = String.raw`#(?:x[a-fA-F\d]+|\d+)(?:;|\b)`;
```

Comment:
THis is just a question.
Before the PR, it uses `\w`, but now `a-fA-F`.
Why do we need to handle only A to F?

Reply:
I think we need use A to F and a to F. Because this regular expression only match hexadecimal String.Because of hexadecimal chars only A to F and a to f, It will occur match error if using \w.

eg, Using \w will match &#x9hello as hexadecimal html String and the  test unit 'hex number with other character 4'  will error.